docs/Proposing-a-new-module.md

```diff
@@ -18,10 +18,10 @@ Proposed or upcoming WordPress core features are also called “[feature project
 
 It is important to understand and think through these stages when proposing a module, as getting a module into the Performance Lab plugin should be a temporary state that is a stepping stone to getting the feature into WordPress core. Every proposed module should have at least one defined owner that takes responsibility for eventually moving the module forward.
 
-It is understandable that this can feel like quite the tedious task - however, rest assured that the rest of the performance team is there to help. If you have never posted anything on the Make WordPress Core blog, don’t worry - there are people on the team who can get you access and review your post drafts. If you have never contributed to the WordPress core codebase, don’t worry - there are people on the team who can review your code for core best practices. If you have never committed anything to WordPress core, don’t worry - there are people on the team who can do that for you. You can always ask any questions you have in the #performance Slack channel or, even better, during the open floor segment of the weekly chats in that channel.
+It is understandable that this can feel like quite the tedious task - however, rest assured that the rest of the Performance Team is there to help. If you have never posted anything on the Make WordPress Core blog, don’t worry - there are people on the team who can get you access and review your post drafts. If you have never contributed to the WordPress core codebase, don’t worry - there are people on the team who can review your code for core best practices. If you have never committed anything to WordPress core, don’t worry - there are people on the team who can do that for you. You can always ask any questions you have in the #performance Slack channel or, even better, during the open floor segment of the weekly chats in that channel.
 
 ## Requirements for a new module
-Based on the above lifecycle for a module, there are a few initial requirements for proposing a new module. The performance team would like to ensure that modules inside the Performance Lab plugin keep moving forward over time, and the requirements outlined here help set the foundation.
+Based on the above lifecycle for a module, there are a few initial requirements for proposing a new module. The Performance Team would like to ensure that modules inside the Performance Lab plugin keep moving forward over time, and the requirements outlined here help set the foundation.
```

Comment:
Can we use lowercase text for `Performance Team` as we used before?

Reply:
@mukeshpanchal27 I think having it lowercase before was actually an oversight; we should always use the capital case Performance Team (or previously, Performance Group).

Reply:
OK, Thanks for the clerification.